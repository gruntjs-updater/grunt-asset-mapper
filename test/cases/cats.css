body {
  background: url('../assets/cat1.jpg');
}

.cat-rap {
  background: url('../assets/unknowncat.jpg');
}