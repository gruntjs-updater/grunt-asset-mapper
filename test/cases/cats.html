<html>
<head>
  <title>Crazy cat test</title>
</head>
<body>

  <div class="cat-rap">
    <img src="../assets/cat1.jpg">
    <img src="../assets/cat2.jpg">
    <img src="../assets/cat3.jpg">
  </div>

</body>
</html>