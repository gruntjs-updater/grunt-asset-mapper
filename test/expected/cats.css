body {
  background: url('../assets/cat1.3a5521.jpg');
}

.cat-rap {
  background: url('../assets/unknowncat.jpg');
}