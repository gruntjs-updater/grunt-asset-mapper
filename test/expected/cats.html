<html>
<head>
  <title>Crazy cat test</title>
</head>
<body>

  <div class="cat-rap">
    <img src="../assets/cat1.3a5521.jpg">
    <img src="../assets/cat2.5cedc3.jpg">
    <img src="../assets/cat3.b998a9.jpg">
  </div>

</body>
</html>